_episodes/02-filedir.md

```diff
@@ -752,11 +753,11 @@ For example, `ls -s` will display the size of files and directories alongside th
 while `ls -S` will sort the files and directories by size, as shown below:
 
 ~~~
-$ ls -s Desktop/shell-lesson-data/data
+$ ls -s ~/Desktop/shell-lesson-data/data
```

Comment:
This should work either with or without the tilde. I don't feel particularly strongly one way or the other, but this code block mixes input and output so it would be good to separate the commands from the output.

Reply:
Sorry, a controversial one found its way in.

From what I can see, the learner is in `~/Desktop/shell-lesson-data` at this point, or possibly `~/Desktop/shell-lesson-data/creatures`. The `ls Desktop/shell-lesson-data/data` command only works if the learner is in their home directory, so I added the tilde so it didn't matter where they were.

But since the learner isn't actually asked to run those commands themselves, I'm happy to revert these changes to keep the code block clean.

Reply:
I must be having a dim day, I just saw what you meant. I can split the code up, no problem.

Reply:
Yes, you're right. My bad.

Reply:
No worries - I've had a few of those lately :)
After a quick review, it's not certain which directory the learner will be in at this point, so maybe a `cd` would be helpful. Either here, or before the next `Nelle` section.
But yes the main point I wanted to make was about splitting input and output blocks.

Reply:
How would you feel about this?

```bash
$ cd ~/Desktop/shell-lesson-data
$ ls -s data
```
```bash
$ ls -S data
```

If they have gone off somewhere else in the meantime and run this, they are back in the right place for the `Nelle` section.

Reply:
Looks great!